### PR TITLE
Update src/main/java/org/getspout/spout/SpoutPlayerListener.java

### PR DESCRIPTION
**Pull request by skyforce77**
Make CustomItems unstackable at pickup or drop

### CODE REVIEW

**Comment by stephenwilliams** [comment]
The formatting D:

File: src/main/java/org/getspout/spout/SpoutPlayerListener.java [hunk 1]
```diff
@@ -254,6 +254,32 @@ public void onPlayerQuit(PlayerQuitEvent event) {
...
 	}
+	@EventHandler(priority = EventPriority.LOWEST)
+	public void onPlayerDrop(PlayerDropItemEvent e) {
+		SpoutItemStack sis = new SpoutItemStack(e.getItemDrop().getItemStack());
+		if(!sis.containsEnchantment(SpoutEnchantment.UNSTACKABLE) && sis.isCustomItem())
+		{
+			CustomItem ci = (CustomItem)sis.getMaterial();
+			if(!ci.isStackable())
+			{
+				sis.addEnchantment(SpoutEnchantment.UNSTACKABLE, 1000);
```
Comment (Pamelloes) [inline src/main/java/org/getspout/spout/SpoutPlayerListener.java:265]: Use the materials counter, not a fixed value.

**Comment by olliestanley** [comment]
OMG @Zidane this is not signed off you now cannot change SpoutPlugin to another license without this guys permission! :P

**Comment by lukespragg** [comment]
DziNeIT, we'd have to get their permission either way. Signing off just means they agree to the license.

This really shouldn't have been pulled without being signed off though, and with the formatting being incorrect.

**Comment by olliestanley** [comment]
OMG this guy doesn't agree to the license I don't even know if there are any downsides to that for you! :P

**Pull request merged**